pkg/debug/checkups/checkups.go

```diff
@@ -250,17 +250,18 @@ func RunDoctor(ctx context.Context, k types.Knapsack, w io.Writer) {
 	}
 }
 
-type runtimeEnvironmentTyp string
+type RuntimeEnvironmentType string
 
 const (
-	StandaloneEnviroment runtimeEnvironmentTyp = "standalone"
-	InSituEnvironment    runtimeEnvironmentTyp = "in situ"
+	StandaloneEnviroment RuntimeEnvironmentType = "standalone"
+	InSituEnvironment    RuntimeEnvironmentType = "in situ"
 )
 
-func RunFlare(ctx context.Context, k types.Knapsack, flareStream io.Writer, runtimeEnvironment runtimeEnvironmentTyp) error {
+func RunFlare(ctx context.Context, k types.Knapsack, flareStream io.WriteCloser, runtimeEnvironment RuntimeEnvironmentType) error {
 	flare := zip.NewWriter(flareStream)
 	defer func() {
 		_ = flare.Close()
+		_ = flareStream.Close()
```

Comment:
nice

Reply:
We could also manually close them, and return an error if the stream close fails. But 🤷

Reply:
doing the manual close now so we get errors